recipes/dfvue/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dfvue" %}
+{% set version = "0.992" %}
+{% set sha256 = "9c50079af9cb6c65d38daad6b939b6690769a8269a14c69c4aa8fab04e847113" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dfvue-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools >=42
+    - setuptools_scm >=3.5
+  run:
+    - python
+    - numpy
+    - matplotlib-base
+    - pandas
+    - tk  # [linux]
```

Comment:
It can probably be a OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies